### PR TITLE
users: wording, grammar

### PR DESCRIPTION
**Pull request by Zyzto**
After review:

* Changed wording for some sentences
* Fixed hamza in some places
* Words order

### CODE REVIEW

File: users.txt [hunk 1]
```diff
@@ -4,16 +4,16 @@ Click to select PNG file=إضغط لإختيار صورة
 Custom image=صورة خاصة
```
Comment (alhamesrami) [inline users.txt:4]: تخصيص الصورة

File: users.txt [hunk 2]
```diff
@@ -4,16 +4,16 @@ Click to select PNG file=إضغط لإختيار صورة
...
 Do not use reserved names=لا تستخدم أسماء محجوزة
-Drag-n-drop a PNG file or click the image at the left=اسحب و ضع ملف صورة او اضغط على الصورة اليسرى
-Edit User=تعديل مستخدم
-Invalid user name=اسم مستخدم غير صحيح
-Only PNG images are allowed=تقبل فقط ملفات صور بصيفة **PNG**
-Password too long=كلمة المرورة طويلة
+Drag-n-drop a PNG file or click the image at the left=اسحب و ضع ملف صورة او اضغط على الصورة على اليسار
+Edit User=تحرير المستخدم
+Invalid user name=اسم المستخدم غير صحيح
+Only PNG images are allowed=فقط صور بصيفة **PNG**
```
Comment (alhamesrami) [inline users.txt:10]: فقط صور بصيغة **PNG**

File: users.txt [hunk 3]
```diff
@@ -4,16 +4,16 @@ Click to select PNG file=إضغط لإختيار صورة
...
+Password too long=كلمة المرورة طويلة جداٌ
```
Comment (alhamesrami) [inline users.txt:11]: كلمة المرور طويلة جدًا

File: users.txt (hunk 2 above)
Comment (Zyzto) [reply users.txt:10]: Do you mean to remove the Bold symbols?

edit: I didn't see that ف the font was too small :D

Comment (alhamesrami) [reply users.txt:10]: Bold symbols is up to our programmer. I don't mind as it meaningful

Comment (spencerjunraid) [reply users.txt:10]: If possible, please keep the "**" "*" formatting on specific words.

**Pull request merged**